setup.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+setup(
+    name="mail_room",
+    description="Mail Room Madness Assignment.",
+    version=0.1,
+    author="David Banks and Jeff Russel",
+    author_email="crashtack@gmail.com, jefferyrayrussel@gmail.com",
+    license='MIT',
+    py_modules=['mail_room'],
```

Comment:
There is no module "mail_room.py" in your src directory.  What does this refer to?